CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 - [`Breaking`] Change any generate transaction function to return `SimpleTransaction` or `MultiAgentTransaction` instance
 - Adds `getUserTransactionHash` which can generate a transaction hash after signing, but before submission
 - Add function to create resource address locally
+- Add `getAccountAllTransactionVersions` which returns versions of all transactions related to the given address.
```

Comment:
These 2 should be under a new dated release version

Reply:
fixed